.editorconfig

```diff
@@ -0,0 +1,31 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
```

Comment:
In Spring Boot, [we've restrict ourselves to XML and Java](https://github.com/spring-projects/spring-boot/blob/master/.editorconfig). We could add groovy and kt files here but I am not keen to add more, at least for a first revision.